media/static-bundles.json

```diff
@@ -820,6 +820,7 @@
     },
     {
       "files": [
+        "js/libs/mozilla-traffic-cop.js",
```

Comment:
Traffic Cop should go in the `<head>` of the page via the `{% experiments %}` block, and use a `switch()` so we can toggle the experiment on/off independently of a production push. See: https://bedrock.readthedocs.io/en/latest/abtest.html#id4